src/components/post/style.css

```diff
@@ -0,0 +1,64 @@
+.post {
+    display: grid;
+    row-gap: 20px;
+}
+
+#post-details {
```

Comment:
Was there a reason for the id, maybe functionality later? Just curious since I assume styling should always be on classes, and it doesn't fit with the rest of the stylesheet